src/haz3lweb/app/input/Shortcut.re

```diff
@@ -251,6 +251,13 @@ let shortcuts = (sys: Util.Key.sys): list(t) =>
       "Run Benchmark",
       Benchmark(Start),
     ),
+    mk_shortcut(
+      ~mdIcon="bolt",
+      ~section="Refactoring",
+      ~hotkey=Keyboard.meta(sys) ++ "+i",
```

Comment:
This is still doing ctrl on my mac

Reply:
Fixed in https://github.com/hazelgrove/hazel/pull/1416/commits/0eabe893237b64e29a108ac0c5691c277c79c496. Was broken for all shortcuts.